onnxscript/converter_test.py

```diff
@@ -615,14 +616,25 @@ def inc_alpha_expanded(A: FLOAT[...], alpha: int) -> FLOAT[...]:
     def test_none_attribute(self):
         @script()
         def explicit_none(X):
-            return op.Squeeze(X, axes=None)
+            return op11.Squeeze(X, axes=None)
 
         @script()
         def implicit_none(X):
-            return op.Squeeze(X)
+            return op11.Squeeze(X)
 
         self.assertSame(explicit_none, implicit_none)
 
+    def test_input_and_attr1(self):
```

Comment:
I would clarify the behavior being tested in the test name. e.g. test_<behavior>